packages/ember-testing/lib/helpers/find_with_assert.js

```diff
@@ -17,12 +17,12 @@
   ```
 
   @method findWithAssert
-  @param {String} selector jQuery selector string for finding an element within
+  @param {String} selector css selector string for finding an element within
   the DOM
-  @param {String} [context] (optional) jQuery selector that will limit the
+  @param {String} [context] (optional) css selector that will limit the
   selector argument to find only within the context's children
-  @return {Object} jQuery object representing the results of the query
-  @throws {Error} throws error if jQuery object returned has a length of 0
+  @return {Object} DOM element representing the results of the query
```

Comment:
This has yet to be implemented.  `find` is still using jquery.  If we merge w/o the `find` change, I can revert these files.